# ros_control関連の設定
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 125
arm_controller:
  type: "position_controllers/JointTrajectoryController"
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
mycobot_arm_controller:
  type: "position_controllers/JointTrajectoryController"
  joints:
    - mycobot_arm_joint_0
    - mycobot_arm_joint_1
    - mycobot_arm_joint_2
    - mycobot_arm_joint_3
    - mycobot_arm_joint_4
    - mycobot_arm_joint_5

